packages/css/src/components/figure/figure.scss

```diff
@@ -0,0 +1,33 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+@use "../../common/text-rendering" as *;
+
+@mixin reset-figure {
+  margin-block: 0;
+  margin-inline: 0;
+}
+
+.ams-figure {
+  display: flex;
+  flex-direction: column;
+  gap: var(--ams-figure-gap);
```

Comment:
Do we want to set a fixed gap here? We just removed that from Card because users should be able to define this themselves.

Reply:
I think we can. The content of Figure is more predictable: it’s always an image (or something alike) and a text.

In a Card, the gap is either between an image and a heading, a heading and a paragraph, a paragraph and a date, etc. That’s why we can’t set a gap there that’s suitable for all these combinations.